imagepruner/v1/register.go

```diff
@@ -0,0 +1,41 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	version   = "v1"
+	groupName = "imagepruner.imageregistry.operator.openshift.io"
```

Comment:
since we (i) made this mistake once already...   probably better to put this in the "imageregistry.operator.openshift.io" group and just call the type "pruner".

Reply:
updated in current push (used imagepruner)